CIFAR10/train/apple_resnet9_8v100_pytorch.json

```diff
@@ -0,0 +1,11 @@
+{
+    "version": "v1.0",
+    "author": "Santiago Akle Serrano, Hadi Pour Ansari, Vipul Gupta, Dennis DeCoste",
+    "authorEmail": "sakle@apple.com",
+    "framework": "Pytorch 1.1.0",
+    "codeURL": "https://github.com/apple/ml-cifar-10-faster",
+    "model": "Custom Resnet 9",
+    "hardware": "Tesla V100 * 8 GPU / 32 GB / 40 CPU",
+    "timestamp": "2019-12-11",
+    "misc": {} 
```

Comment:
Can you add details about your training routine (e.g., learning rate schedule, batch size, optimizer, etc.) and versions for any other important libraries (e.g., Cuda)? For example, take a look at [David Page's original submission](https://github.com/stanford-futuredata/dawn-bench-entries/blob/master/CIFAR10/train/davidcpage_resnet9_1v100-ec2_pytorch.json).